### PR TITLE
fixed examples with not annotated target word

### PR DESCRIPTION
**Pull request by Vitaly-Protasov**
Overall, for each file I found those examples which included not aggregated target word by special tags such as `<b><i> `and `</i></b>`.

Still, there are other examples which I not decided to fix, for example:
`Европа начала I в. н.э.`
Target word: `век`

And

`Ионис судорожно вцепился в брандспойт, а я повторил: -- Багира, ап!`
Target word: `апостол`.

Yes, this works only for short form of words `апостол` and `век` :)

### CODE REVIEW

**Comment by Vitaly-Protasov** [comment]
Finally, @akutuzov, you can check changes:)

File: raw_annotations/rusemshift1_raw_annotations.tsv [hunk 1]
```diff
@@ -34,7 +34,7 @@
...
-35	амбиция	Хоть с генеральскими детьми поводиться, так и тем на ногу себе наступить не позволит, потому я первым делом наблюдаю, чтобы в моих детях эта благородная <b><i>амбиция</i></b> была; как сам я, сударыня, моему императору майор, в штаб-офицерском ранге числюсь, так уж и желаю вполне, чтобы мои дети достоинство родительского звания соблюдали.	Он уж, поди, и сам был не рад, что такая идея ему пришла, но команда отдана, отменить -- <b><i>амбиция</i></b> не позволяла.	4.0	4	4	4
+35	амбиция	Хоть с генеральскими детьми поводиться, так и тем на ногу себе наступить не позволит, потому я первым делом наблюдаю, чтобы в моих детях эта благородная <b><i>амбиция</i></b> была; как сам я, сударыня, моему императору майор, в штаб-офицерском ранге числюсь, так уж и желаю вполне, чтобы мои дети достоинство роодимое препятствия для танков и моточастей пр-ка.дительского звания соблюдали.	Он уж, поди, и сам был не рад, что такая идея ему пришла, но команда отдана, отменить -- <b><i>амбиция</i></b> не позволяла.	4.0	4	4	4
```
Comment (akutuzov) [inline raw_annotations/rusemshift1_raw_annotations.tsv]: Why a seemingly random piece of text is added here?

Comment (akutuzov) [reply raw_annotations/rusemshift1_raw_annotations.tsv]: Can you fix this and make sure this does not happen elsewhere?

**Review by akutuzov — CHANGES_REQUESTED** [review]

**Comment by Vitaly-Protasov** [comment]
@akutuzov  Fixed. Recheked code and other sentences. Bag was fixed

**Review by akutuzov — APPROVED** [review]
Thanks a lot for your help!

**Pull request merged**